src/pages/BrowseUploads/LicenseBrowser/index.jsx

```diff
@@ -117,20 +117,20 @@ const LicenseBrowser = () => {
               <tbody>
                 {filesData &&
                   filesData.length > 0 &&
-                  filesData.map((data, index) => (
+                  filesData.map((data) => (
                     <>
-                      {index < 10 ? (
-                        <tr key={data.id}>
-                          <td>{data.filePath}</td>
-                          <td>{data.findings.scanner}</td>
-                          <td>-</td>
-                          <td>-</td>
-                          <td>-</td>
-                          <td>-</td>
-                        </tr>
-                      ) : (
-                        ""
-                      )}
+                      <tr key={data.id}>
+                        <td>{data.filePath}</td>
+                        <td>{data.findings.scanner}</td>
```

Comment:
If `findings.scanner` is an array, the output is not pleasing
![image](https://user-images.githubusercontent.com/18077542/183417644-ccc9144e-12f6-47c4-a757-80579096eb1e.png)
![image](https://user-images.githubusercontent.com/18077542/183417708-4e01a2c6-9a70-4a38-b47c-bcdb7d388b4e.png)


As you can see, the UI shows `MITMIT` which is incorrect.
1. In such cases, it should show `LIC1, LIC2`.
2. The output should show unique values.
    - `["MIT","MIT","Apache-2.0"] => MIT, Apache-2.0`

Reply:
Same applies for `findings.conclusion`

Reply:
Resolved